roaster-bom/pom.xml

```diff
@@ -11,11 +11,12 @@
 
 	<properties>
 		<version.org.eclipse.microprofile>6.0</version.org.eclipse.microprofile>
-		<version.hu.icellmobilsoft.coffee>2.3.0</version.hu.icellmobilsoft.coffee>
+		<version.hu.icellmobilsoft.coffee>2.7.0-SNAPSHOT</version.hu.icellmobilsoft.coffee>
 		<version.org.junit>5.9.1</version.org.junit>
 		<version.testcontainers>1.17.6</version.testcontainers>
 		<version.mockserver-client-java>5.14.0</version.mockserver-client-java>
 
+		<version.org.apache.commons.commons-lang3>3.14.0</version.org.apache.commons.commons-lang3>
```

Comment:
Ezzel óvatosan majd, mert a wildflyba (még a 30ban is) 3.11 van.

Reply:
Roaster szemszögéből sztem nem számít a wildfly.